apps/storybook/src/SharedResearchList.stories.tsx

```diff
@@ -1,31 +1,46 @@
-import React from 'react';
-
+import React, { ComponentProps } from 'react';
 import { SharedResearchList } from '@asap-hub/react-components';
-import { text, number, select } from '@storybook/addon-knobs';
-import { createResearchOutputResponse } from '@asap-hub/fixtures';
+import { boolean, number } from '@storybook/addon-knobs';
+import { StaticRouter } from 'react-router-dom';
 
 export default {
-  title: 'Organisms / Shared Research / List',
+  title: 'Templates / Shared Research / List',
+};
+
+const researchOutput: Omit<
```

Comment:
y u no fixture?

Reply:
Oh I guess probably because it was a copy paste move thingy

Reply:
This entire PR is moving things around. Not going to address that kind of problem